utils/ci-test.sh

```diff
@@ -1,31 +1,42 @@
 #!/bin/bash
 
+# SKIP_LINT=true - only run the tests
+
 set -e
 
+MATRIX=(
+	""
+	"-tags coprocess"
+)
+
 # print a command and execute it
 show() {
-	echo "$@"
+	echo "$@" >&2
```

Comment:
It depends - I usually only use stdout for useful output, so not log messages even if they are not errors :) Otherwise, pipes would be of little use.

Travis doesn't treat the text differently either, see https://travis-ci.org/TykTechnologies/tyk/jobs/193369280#L304

I don't see a clear winner, so I went with this for consistency.